apps/vth-frontend/src/components/Card/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { Heading2, Paragraph } from '@utrecht/component-library-react';
+import clsx from 'clsx';
+import Image from 'next/image';
+import Link from 'next/link';
+import React, { FC, HTMLAttributes } from 'react';
+import './index.style.css';
+
+interface CardProps extends Omit<HTMLAttributes<HTMLDivElement>, 'children'> {
+  alt: string;
+  body: string;
```

Comment:
Could you please rename the 'body' property to 'description' and make sure it aligns with the Strapi API?